community/create-a-kvrocks-controller-release.md

```diff
@@ -0,0 +1,210 @@
+
+    This is a call for a vote to release Apache Kvrocks Controller version ${release_version}.
+
+    The tag to be voted on is ${release_version}-rc${candidate_number}.
+
+    The release candidate:
+
+    https://dist.apache.org/repos/dist/dev/kvrocks/kvrocks-controller/${release_version}/
+
+
+    Keys to verify the release candidate:
+
+    https://downloads.apache.org/kvrocks/KEYS
+
+
+    Git tag for the release:
+
+    https://github.com/apache/kvrocks-controller/releases/tag/${release_version}-rc${candidate_number}
+
+    Please download, verify, and test.
+
+    The VOTE will remain open for at least 72 hours.
+
+    [ ] +1 approve
+    [ ] +0 no opinion
+    [ ] -1 disapprove with the reason
+
+    To learn more about apache kvrocks, please see
+    https://kvrocks.apache.org/
+
+    Checklist for reference:
+
+      [ ] Download links are valid.
+      [ ] Checksums and signatures.
+      [ ] LICENSE/NOTICE files exist
+      [ ] No unexpected binary files
+      [ ] All source files have ASF headers
+      [ ] Can compile from source
+      [ ] All Tests Passed
+
+      More detailed checklist  please refer to:
+      https://kvrocks.apache.org/community/verify-a-release-candidate
+
+Thanks
+```
+
+After at least 72 hours with at least 3 +1 binding vote (from Kvrocks PMC member) and no veto, claim the vote result:
+
+```text
+[RESULT][VOTE] Release Apache Kvrocks Controller ${release_version}
+
+The vote to release Apache Kvrocks Controller ${release_version} has passed.
+
+The vote PASSED with xxx binding +1 and 0 -1 votes:
+
+Binding votes:
+- Yuan Wang
+- tison
+- hulk
+- Liang Chen
+- Jean-Baptiste Onofré
+- Xiaoqiao He
+- donghui liu
+
+No Binding votes:
+- XXX
+- XXX
+- XXX
+
+Vote thread:
+
+${vote_thread_url}
+
+Thank you to all the above members to help us to verify and vote for the
+${release_version} release.
+
+Thanks
+```
+
+## Officially released
+
+### Publish artifacts to SVN RELEASE branch
+
+```shell
+svn mv https://dist.apache.org/repos/dist/dev/kvrocks/kvrocks-controller/${release_version} https://dist.apache.org/repos/dist/release/kvrocks/kvrocks-controller/${release_version} -m "Release ${release_version}"
+```
+
+### Publish release note
+
+Edit the title of the draft release note and use the new ${release_version} tag, which github will create automatically.
+
+Then, publish release.
+
+### Update website links
+
+Update [releases data file](https://github.com/apache/kvrocks-website/blob/main/src/components/Releases/index.tsx).
+
+### Send the announcement
+
+Send the release announcement to `dev@kvrocks.apache.org` and CC `announce@apache.org`:
+
+> You have to use `${name}@apache.org` post email to `announce@apache.org`
+
+```html
+[ANNOUNCE] Release Apache Kvrocks Controller ${release_version}
+
+Hi all,
+
+The Apache Kvrocks community is pleased to announce
+that Apache Kvrocks Controller ${release_version} has been released!
+
+Kvrocks is a distributed key value NoSQL database that uses RocksDB
```

Comment:
Introduction for kvrocks controller?

Reply:
Sure, will change to Kvrocks controller introduction and add the download page.